docs/source/guide/dataset_create.md

```diff
@@ -20,26 +20,217 @@ hide_sidebar: true
 
+## Datasets using AWS
+
+Requirements:
+
+- Your data is located in an AWS S3 bucket.
+- You have an AWS access key with view permissions for the S3 bucket. 
+- Your AWS S3 bucket has CORS configured properly. Configuring CORS allows you to view the data in Label Studio. When CORS is not configured, you are only able to view links to the data. 
+
+{% details <b>Configure CORS for the AWS S3 bucket</b> %}
+
+**Prerequisites:**
+
+You have edit access to the bucket. 
+
+###### Configure CORS access to your bucket
+
+Set up cross-origin resource sharing (CORS) access to your bucket using a policy that allows GET access from the same host name as your Label Studio deployment. 
+
+You can use the AWS Management Console, the API, or SDKs. For more information, see [Configuring cross-origin resource sharing (CORS)](https://docs.aws.amazon.com/AmazonS3/latest/userguide/enabling-cors-examples.html) 
+
+You can use or modify the following example:
+
+```shell
+[
+    {
+        "AllowedHeaders": [
+            "*"
+        ],
+        "AllowedMethods": [
+            "GET"
+        ],
+        "AllowedOrigins": [
+            "*"
+        ],
+        "ExposeHeaders": [
+            "x-amz-server-side-encryption",
+            "x-amz-request-id",
+            "x-amz-id-2"
+        ],
+        "MaxAgeSeconds": 3000
+    }
+]
+```
+
+{% enddetails %}
+
+{% details <b>Create an AWS access key</b> %}
+
+
+**Prerequisites:**
+
+- You must have the admin permissions in your AWS account to generate keys and create service accounts. 
+
+###### Create a policy for the user
+
+You need a permissions policy that can list S3 buckets and read objects within buckets. If you already have a policy that does this, or if you feel comfortable using the pre-configured **AmazonS3ReadOnlyAccess** policy, then you can skip this step. 
+
+1. From the AWS Management Console, use the search bar or navigation menu to locate the **IAM** service.
+2. Select **Access Management > Policies** from the menu on the left.
+3. Click **Create policy**. 
+4. From the policy editor, select the **JSON** option and paste the following: 
+
+    ```json
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
+            "Action": [
+                "s3:ListBucket",
+                "s3:GetObject"
+            ],
+            "Resource": "*"
+        }
+    ]   
+}
+    ```
+
+    If you want to further restrict the permissions to certain buckets, edit the `Resource` key as follows:
+
+    ```json
+
+           "Resource": [
+                "arn:aws:s3:::<your_bucket_name>",
+                "arn:aws:s3:::<your_bucket_name>/*"
+            ]
+
+    ```
+
+
+###### Create a user 
+
+This user can be tied to a specific person or a group. 
+
+1. From the AWS Management Console, use the search bar or navigation menu to locate the **IAM** service.
```

Comment:
It probably makes sense to have our own steps here that are specific to the process for LS, but is there an obvious page in AWS's docs to link to here? So when our steps here go stale a user can still fall back to the AWS docs.

Reply:
Yeah, I like to make it easier for beginner users so they don't have to flip back and forth between two sets of docs. Also the AWS docs can be overwhelming. But good point about it going stale -- added a link list at the top to AWS docs.